com.mattermost.Desktop.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "com.mattermost.Desktop",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "18.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "mattermost-flatpak",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--talk-name=ca.desrt.dconf",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "mattermost-desktop",
+            "buildsystem": "simple",
+            "build_commands": [
+                "mkdir -p /app/main /app/bin",
+                "cp -ra main/* /app/main/",
+                "install mattermost.sh /app/bin/mattermost-flatpak",
+                "install -Dm644 com.mattermost.Desktop.desktop /app/share/applications/com.mattermost.Desktop.desktop",
+                "install -Dm644 com.mattermost.Desktop.appdata.xml /app/share/appdata/com.mattermost.Desktop.appdata.xml",
+                "install -Dm644 /app/main/icon.png /app/share/icons/hicolor/512x512/apps/com.mattermost.Desktop.png"
+            ],
+            "sources" : [
```

Comment:
You should add the source tarball so it gets exported to `Source` extension.

Reply:
Could you please elaborate more on what you're saying? Preffereably with an example. It's my first flatpak, so I appreciate your patience ;)

Reply:
What I meant by that is that you could add the [link](https://github.com/mattermost/desktop/archive/v4.1.2.tar.gz) to the source tarball so it gets exported to `Source` extension, something like:

```json
{
  "type": "file",
  "26ac290aec7daef5d75fd0e5c1027ad8bc34e7cadb03cda8408f0585bc27feb8",
  "https://github.com/mattermost/desktop/archive/v4.1.2.tar.gz"
},
```